_release-pypi/action.yml

```
@@ -96,6 +96,13 @@ inputs:
     required: true
     type: boolean
 
+  use-uv:
+    description: |
+      Whether to use uv as the default package manager instead of pip. Default value is ``true``.
+    default: true
```

```suggestion
    default: false
```
I would default to `false` for the actions were you didn't implement `uv` initially:
- _release-pypi
- build-library
- doc-changelog
- hk-package-clean-except
- hk-package-clean-untagged

@jorgepiloto @RobPasMue What do you think ?

If this is validated, the current change needs to be applied in other actions

---

Thanks for raising this, @SMoraisAnsys. Agreed. If we proceed implementing `uv` everywhere, then at least let's disable it for "critical" actions.

---

I completely agree to , thanks @SMoraisAnsys and @moe-ad!

---

@SMoraisAnsys @jorgepiloto @RobPasMue 
Implemented. `ansys/actions/build-wheelhouse` was also added.

---

> Thanks for raising this, @SMoraisAnsys. Agreed. If we proceed implementing `uv` everywhere, then at least let's disable it for "critical" actions.

Did you mean "non critical" actions ? My idea was to follow your thoughs on having a treshold to default to `uv` or not. Like "if the number of packages installed is >2" then "true" else "false".

---

Oh! I thought we were discretizing based on actions that build, release and deploy "uv: false" and the rest "uv: true".